modules/intentIqAnalyticsAdapter.js

```diff
@@ -7,6 +7,10 @@ import { config } from '../src/config.js';
 import { EVENTS } from '../src/constants.js';
 import { MODULE_TYPE_ANALYTICS } from '../src/activities/modules.js';
 
+// Since IntentIQ analytics module work based on ID module configurations, there is shared code
+// eslint-disable-next-line prebid/validate-imports
+import { detectBrowser } from './intentIqIdSystem.js';
```

Comment:
you need to import from a shared library, not from another module. You can put it in /libraries/...

Reply:
Done